gulpfile.js

```diff
@@ -82,6 +83,58 @@ function createReloadServer() {
   });
 }
 
+function writeServiceWorkerFile() {
+  var path = require('path');
+  var rootDir = 'dist';
+
+  var config = {
+    cacheId: 'polymerproject',
+    staticFileGlobs: [
+      rootDir + '/1.0/index.html',
```

Comment:
Why not? When `templates/head-meta.html` changes, on the next rebuild (because you need it to deploy) this will regenerate `dist/1.0/index.html`, because we regenerate everything on every build. And this new, regenerated file will be different.